support/k8s/k8s-workload-registrar/config.go

```diff
@@ -68,6 +69,9 @@ func ParseConfig(hclConfig string) (*Config, error) {
 	if c.Cluster == "" {
 		return nil, errs.New("cluster must be specified")
 	}
+	if c.PodLabel != "" && c.PodAnnotation != "" {
+		return nil, errs.New("workload registration mode specification is incorrect")
```

Comment:
maybe a more specific message? something like `cannot specify both pod_label and pod_annotation`?

Reply:
I fixed it with https://github.com/spiffe/spire/pull/1047/commits/79f65711f6b93943e6ca16b8b02c1e4c4cd39b6f .